### PR TITLE
Persisting notifs

### PR DESCRIPTION
**Pull request by Tyler-Ouellette**
Notifications working, not totally persisting though. I can make them persist, but without marking them as read in the db, it just grabs all and keeps adding them in the notification dropdown so i commented out the getNotifications in app.vue and left it as it was before hand.

### CODE REVIEW

File: resources/js/bootstrap.js [hunk 1]
```diff
@@ -26,18 +26,18 @@ if (token) {
...
+ * */
 
 import Echo from 'laravel-echo';
 
 window.Pusher = require('pusher-js');
 
 window.Echo = new Echo({
     broadcaster: 'pusher',
-    key: '118c0f9922364667dc61',
+    key: '47cde7e5985f5f95aac3',
```
Comment (Tyler-Ouellette) [inline resources/js/bootstrap.js]: this is my key  so that might need  changing for production

Comment (Tyler-Ouellette) [reply resources/js/bootstrap.js]: @James-N-M

**Comment by James-N-M** [comment]
Is there anyway you can split this into multiple pull requests, like one thing a pull request just so its not such a big change and we can merge and test as we go

**Comment by James-N-M** [comment]
because we have different ides every file you touch gets a change and its reall hard to see whats actually new and whats old, I cant merge till we break this out or it could result in something i can't debug

**Comment by James-N-M** [comment]
also if you see something style wise that's different than something you'd prefer just leave it for the reviewers sake less red the easier to review, instead you would maybe wanna do that in a separate pr. 

PR Example: ```Added New User Event```
then the code that only has to do with the New User Event,

also I noticed your using the same event for multiple things, if thats the case it might make more sense to make a more generic event and channel for "System" things and same event object, different messages.

**Pull request merged**